### ISSUE TITLE
Create new image carousel component for homepage.

### ISSUE THREAD

**Issue by shikha-16** [shikha-16/Women-in-Technology#134]
Create new image carousel component for homepage, with clearer images and better design.

**Comment by logicalahara** [comment]
I can help you with carousel redesign, but I'll require some help  for designing images

**Comment by shikha-16** [comment]
The main job would be to create the images @logicalahara , this is a graphic design issue. If you can make the images on your own I will assign this to you. Please let me know.

**Comment by logicalahara** [comment]
4 images to design but using the current carousel on site. Is that right ?

**Comment by logicalahara** [comment]
I'm on it, please give me some light on key elements to include in design

**Comment by shikha-16** [comment]
> 4 images to design but using the current carousel on site. Is that right @shikha-16 ?

Yeah, kind of. We could get on a call to discuss this, please join the discord!

**Comment by abhishek-kumar-91** [comment]
I want to work on this issue, Can you assign me?

**Comment by ayodhyaw2002** [comment]
Can someone assign me
